examples/sveltekit-pwa/pwa-configuration.js

```diff
@@ -1,10 +1,12 @@
+const mode = process.env.SOURCE_MAP === 'true' ? 'development': undefined
+
```

Comment:
could the `if (selfDestroying)` statement at the bottom of this file be removed and the `selfDestroying` varibale be inlined such that you just always do:
```
pwaConfiguration.selfDestroying = process.env.SW_DESTROY === 'true'
```

Reply:
moved to the manifest entry